src/helpers/utils.ts

```diff
@@ -1,6 +1,14 @@
 import providers, { fallbackProvider } from "../providers";
 import { IProviderInfo, IInjectedProvidersMap } from "./types";
 
+export function getProviderPackage(name: string) {
```

Comment:
Maybe call this `importProviderPackage`, to be clear what it does and to avoid confusion with providers/index/providerPackages ?